lib/browser/property-descriptor.ts

```diff
@@ -265,21 +265,19 @@ export function propertyDescriptorPatch(api: _ZonePrivate, _global: any) {
     return;
   }
 
-  const supportsWebSocket = typeof WebSocket !== 'undefined';
+  const supportsWebSocket = typeof WebSocket !== o;
   if (canPatchViaPropertyDescriptor()) {
     const ignoreProperties: IgnoreProperty[] = _global.__Zone_ignore_on_properties;
     // for browsers that we can patch the descriptor:  Chrome & Firefox
     if (isBrowser) {
+      const w: any = window;
```

Comment:
local vars get renamed. Calling it `w` makes it hard to read. Could we call it `window` and that way we don't have to change the code bellow? I know `window = window` would not work, but something like that?

Reply:
yes, you are right, I have changed the variable to `internalWindow`.